tools/Microsoft.TemplateSearch.TemplateDiscovery/Program.cs

```diff
@@ -7,10 +7,10 @@ namespace Microsoft.TemplateSearch.TemplateDiscovery
 {
     internal class Program
     {
-        private static async Task Main(string[] args)
+        private static async Task<int> Main(string[] args)
```

Comment:
@GangWang01 is it fix for return no error on test failures?

Reply:
I think so. At least the tool should return exit code from https://github.com/dotnet/templating/blob/db350601d79760fd7a32e36cdefad48c9d804f56/tools/Microsoft.TemplateSearch.TemplateDiscovery/TemplateDiscoveryCommand.cs#L108-L142 to bash.